helpers/getReadingTime.js

```diff
@@ -0,0 +1,11 @@
+const getReadingTime = text => {
+  const readingMinutes = Math.ceil(text.split(' ').length / 265);
+  const readingTime = readingMinutes => {
+    const hours = Math.floor(readingMinutes / 60);
+    const mins = readingMinutes % 60;
+    return `${hours}h ${mins}min`.replace('0h ', ''); // if we have 0 hours remove it.
```

Comment:
One line comment is not very much needed

Reply:
I want to explain why I am replacing hours with an empty string. It can be confusing